src/Rings/mpoly-ideals.jl

```diff
@@ -642,11 +654,8 @@ Return whether `I` is a radical ideal.
 Computes the radical.
 """
 @attr Bool function is_radical(I::MPolyIdeal)
-  if has_attribute(I, :is_prime) && is_prime(I)
-    return true
-  end
-
-  return I == radical(I)
+  has_attribute(I, :is_prime) && return is_prime(I)
```

Comment:
this change looks wrong to my eyes.
Consider a non-prime, radical ideal. This new version would return `false`, although the ideal is indeed radical

Reply:
True. Thanks for the catch.